content/docs/user-guide/what-is-dvc.md

```diff
@@ -47,3 +47,18 @@ can version experiments, manage large datasets, and make projects reproducible.
 
   > Git servers, as well as SSH and cloud storage providers are supported,
   > however.
+
+## DVC is not Git!
+
+DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
+large data files and directories for versioning (among other
+[purposes](/doc/user-guide/dvc-files-and-directories)). These metafiles change
+along with your data, and you can use Git to place them under
+[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
+as a proxy to the actual data versions, which are stored in the <abbr>DVC
+cache</abbr> (outside of Git). This does not replace features of Git.
+
+DVC does, however, provide several commands that are somewhat analogous to their
```

Comment:
`somewhat analogous` can we replace this with something explicit?

Reply:
e.g. from use perspective?

Reply:
I did list examples below but OK, I'm moving them higher up in the text here.